packages/patternfly-4/react-core/src/components/Alert/Alert.js

```diff
@@ -60,20 +83,32 @@ const Alert = ({
     </React.Fragment>
   );
 
-  const customClassName = css(styles.alert, getModifier(styles, variant, styles.modifiers.info), className);
-
+  const customClassName = css(
+    styles.alert,
+    getModifier(styles, variant, styles.modifiers.info),
+    className,
+    ((toast || ariaLive) && 'pf-m-live')
+  );
   return (
     <div {...props} className={customClassName} aria-label={ariaLabel}>
       <AlertIcon variant={variant} />
-      <h4 className={css(styles.alertTitle)}>{readerTitle}</h4>
-      {children && (
-        <div className={css(styles.alertDescription)}>
-          <p>{children}</p>
-        </div>
-      )}
-      {action && (
-        <div className={css(styles.alertAction, className)}>{React.cloneElement(action, { title, variantLabel })}</div>
-      )}
+      <div
+        className="pf-c-alert__content"
+        aria-live={toast ? 'polite' : ariaLive}
+        aria-relevant={toast ? 'additions text' : ariaRelevant}
+        aria-atomic={toast ? 'false' : ariaAtomic}>
```

Comment:
I just want to throw a suggestion out that would also work, but maybe simplify some of this component...

We could remove these as explicitly defined props and all the logic associated with them, and instead do that thing where the consumer specifies html attributes on the component, and they are passed down to an html element. My personal expectation—which may not be common—is that those cases would add the attributes to the parent html element of the component. If that is the expectation, that would leave us with this:
```
<div class="pf-c-alert pf-m-success pf-m-live" aria-label="Success Alert" aria-live="polite" aria-relevant="additions text" aria-atomic="false">
  <div class="pf-c-alert__icon">...</div>
  <div class="pf-c-alert__content">
    <h4 class="pf-c-alert__title">...</h4>
  </div>
</div>
```

Instead of this:
```
<div class="pf-c-alert pf-m-success pf-m-live" aria-label="Success Alert">
  <div class="pf-c-alert__icon">...</div>
  <div class="pf-c-alert__content" aria-live="polite" aria-relevant="additions text" aria-atomic="false">
    <h4 class="pf-c-alert__title">...</h4>
  </div>
</div>
```

I think this is totally fine. The contents are still announced. It just isn't as clean because ALL the contents are announced, including button labels.